var/spack/repos/builtin/packages/fontconfig/package.py

```diff
@@ -23,6 +23,14 @@ class Fontconfig(AutotoolsPackage):
     depends_on('font-util')
     depends_on('libuuid', when='@2.13.1:')
 
+    def setup_build_environment(self, spack_env):
+        spack_env.set(
+            "FREETYPE_CFLAGS",
+            self.spec['freetype'].headers.include_flags + "/freetype2")
+        spack_env.set(
+            "FREETYPE_LIBS",
+            self.spec['freetype'].libs.ld_flags)
```

Comment:
Minor OCD. Can we call the argument `env` instead of `spack_env`? Since the function takes a single argument `spack_` does not add meaning and the majority of the other packages is already using `env`.

Reply:
@marcmengel Additional request at https://github.com/spack/spack/pull/18113#issuecomment-684699607